views/CurrentVideoView.js

```diff
@@ -0,0 +1,8 @@
+var CurrentVideoView = Backbone.View.extend({
+  template: Handlebars.compile($("#current-video-template").html()),
+
+  render: function () {
+    this.$el.html(this.template(this.model.toJSON()));
+    return this
```

Comment:
I do this all the time, but decide whether or not you're going to use `;` and stick with it